### ISSUE TITLE
U200 infamy icons corrupted waypoint sprites

### ISSUE THREAD

**Issue by JennyXJ9** [HugoZink/PD2_BetterJokers#4]
I believe u200's new infamy card icons resulted in the sprites used by your custom waypoint solution looking corrupted.

MUI's player name icons and TdlQ's own waypoints also had this issue until updating for it, which leads me to believe the same issue is happening here.

The function of the waypoints with regards to ordering jokers still functions as intended so it's just a visual issue.

**Comment by morerokk** [comment]
Yep, I confirmed that this is happening but haven't had time to fix it yet. Thanks for your report!
